src/LabelerServer.ts

```diff
@@ -260,24 +268,23 @@ export class LabelerServer {
 	 * @returns The created labels.
 	 */
 	async createLabels(
-		subject: { uri: string; cid?: string | undefined },
+		subject: LabelSubject,
 		labels: { create?: Array<string>; negate?: Array<string> },
 	): Promise<Array<SavedLabel>> {
 		await this.dbInitLock;
 
-		const { uri, cid } = subject;
 		const { create, negate } = labels;
 
 		const createdLabels: Array<SavedLabel> = [];
 		if (create) {
 			for (const val of create) {
-				const created = await this.createLabel({ uri, cid, val });
+				const created = await this.createLabel({ ...subject, val });
 				createdLabels.push(created);
 			}
 		}
 		if (negate) {
 			for (const val of negate) {
-				const negated = await this.createLabel({ uri, cid, val, neg: true });
+				const negated = await this.createLabel({ ...subject, val, neg: true });
```

Comment:
Prefer to keep the destructuring of subject, just in case any stray properties get in